brigade/core/configuration.py

```diff
@@ -0,0 +1,61 @@
+import ast
+import os
+
+
+import yaml
+
+
+CONF = {
+    'num_workers': {
+        'description': 'Number of Brigade worker processes',
+        'env': 'BRIGADE_NUM_WORKERS',
+        'type': int,
+        'default': 20,
+    },
+    'raise_on_error': {
+        'description': "If set to ``True``, :meth:`run` method of will raise an exception if at "
+                       "least a host failed.",
+        'env': 'BRIGADE_RAISE_ON_ERROR',
+        'type': bool,
+        'default': True,
+    },
+    'ssh_config_file': {
+        'description': 'User ssh_config_file',
+        'env': 'BRIGADE_SSH_CONFIG_FILE',
```

Comment:
should we assume always that is `"BRIGADE_{}".format(key.upper())`?

Reply:
Not really sure, perhaps not for things like http proxy settings and such?

Reply:
maybe we can default it to be as I described but have the possibility to override with this method

Reply:
Yes that makes sense, also makes the dict easier to read. Pushed a change with this.